### ISSUE TITLE
Game breaking bug with burning godzilla

### ISSUE THREAD

**Issue by essentery** [vret001/KU#560]
**Describe the bug**
a bug that gives burning godzilla infinite meltdown and you dont take damage from meltdown

**When did you last experience the issue:**
7/26/2022 8:31 AM

**To Reproduce**
1. Go to Burning godzilla
2. Kill someone while meltdown is active
3. Try a couple times if you dont get it right away

**Expected behavior**
for the meltdown bar to go down and slowly take damage

**Screenshots**
![RobloxScreenShot20220726_083001932](https://user-images.githubusercontent.com/100055786/181006721-09a8d80a-19a5-4925-a157-ea3cc97a24be.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version sorry i dont know

**Additional context**
basically with this bug burning can do 2710 damage around every 4 seconds and 3 shot a dest with beam spam alone 
all the attacks are damage boosted indefinitely
